crates/uv-tool/src/lib.rs

```diff
@@ -204,7 +204,7 @@ impl InstalledTools {
         match PythonEnvironment::from_root(&environment_path, cache) {
             Ok(venv) => {
                 debug!(
-                    "Using existing environment for tool `{name}`: {}",
+                    "Found existing environment for tool `{name}`: {}",
```

Comment:
Note: drive-by change to make debug logs more uniform. After this, the output now looks like this:
```
DEBUG Found existing environment for tool `poetry`: .local/share/uv/tools/poetry
DEBUG Found existing interpreter for tool `poetry`: /home/lucab/.local/share/uv/tools/poetry/bin/python3
```